content/docs/atom-guide/atom-sample-viewer/debug-menu-samples.md

```diff
@@ -11,11 +11,54 @@ This section dives into the debug menu samples demonstrated in the Atom Sample V
 
-![CPU Profiler](/images/atom-guide/atom-sample-viewer/cpu-profiler.jpg)
+To open the CPU Profiler widget, click on `Profile` &rarr; `CPU Profiler` from the ImGui debug menu.
+
+### High-level Toolbar
+
+The CPU profiler widget has two views, a statistics view and a visualizer. These two views share a common header for controlling the profiler.
+
+![CPU Profiler Views](/images/atom-guide/atom-sample-viewer/cpu-profiler-views.png)
+
+* Swap to statistics is used to swap between the statistics view and the visualizer.
+* Resume can be used to start or stop profiling data collection from Atom.
+* Capture takes a single-frame capture of CPU profiling data and saves it to a location on disk, by default into /<project>/user/CpuProfiler/.
+* Begin can be used to start a continuous capture of CPU profiling data, during which Atom is storing many frames of data. End will end the continuous capture and save the data that was collected to disk, by default into /<project>/user/CpuProfiler/.
+* Load File will bring up the file picker window which will list all of the.json files in /<project>/user/CpuProfiler/. This list of files is sorted in decreasing order according to modification time, so the most recent capture is at the top.
+
+### Loading Captures
+
+![CPU Profiler File Picker](/images/atom-guide/atom-sample-viewer/cpu-profiler-file-picker.png)
+
+* Load Selected will load the selected capture into memory, after which the data will be visible in the statistics and visualizer views.
+
+### Statistics
+
+![CPU Profiler Statistics](/images/atom-guide/atom-sample-viewer/cpu-profiler-statistics.png)
+
+The statistics view focuses on processing and displaying the quantitative data collected by Atom's CPU profiler. Each row in the table represents a specific profiling region, and all columns in the table can be sorted by clicking on the column headers:
+* Group: The overall renderer area (ex. RPI, RHI, DX12) that the profiling region occurs in.
+* Region: The lower-level name of the profiling region, often the name of a function call.
+* MTPC: The mean time per call in milliseconds of the profiling region aggregated across all of the data collected while the CPU profiler widget has been active.
+* Max: The maximum execution time of the region in milliseconds during the last frame's execution. For regions that only appeared once in the last frame, this will be equal to the total time.
+* Invocations: The number of times that the profiling region was executed in the last frame.
+* Total: The overall time spent in milliseconds executing this profiling region on the last frame, summing up each invocation of the region.
+
+Rows in the table with a specific group or region name can also be searched for with the filter, which can be easily cleared by clicking  Clear Filter. Reset Table will clear all of the table's saved data and is recommended when switching samples.
+
+### Visualizer
```

Comment:
I have two separate suggestions for this subsection, labeled as **Option A** and **Option B**.

**Option A**: Does not reorder the contents of this subsection. Minimal edit. 
**Option B**: Reformat this subsection into two texts: _conceptual_ and _action_. The conceptual paragraphs describe the visualizer and its components, such as the histogram. The action table groups all of the ways the user can interact with the visualizer to do different functions. 

These options will make more sense as you review the suggestions that follow. I believe Option B organizes the information better, so there's a clear line between "learn" and "do. Also, a user can easily refer to the actions table to recall how to do a specific function.

Reply:
Went with option B, the sections are now informally divided into two parts